tailscale/resource_acl.go

```diff
@@ -36,6 +36,11 @@ func resourceACL() *schema.Resource {
 				ValidateDiagFunc: validateACL,
 				Description:      "The JSON-based policy that defines which devices and users are allowed to connect in your network",
 			},
+			"allow_overwrite": {
```

Comment:
This seems a bit too ambiguous of a name. For example, it's not clear right now that this only applies when a new resource is created.

Any thoughts on using a more specific name? For example, `overwrite_on_creation` or `overwrite_existing_content`?

Reply:
hmm, good point. I just used the one I'm familiar with, from  https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record#allow_overwrite.

I think `overwrite_existing_content` is clearer than `overwrite_on_creation`

Reply:
If you could please rename the attribute to `overwrite_existing_content` I will be happy to merge and cut a new release. Thank you!

Reply:
updated, thanks